### PR TITLE
data:navitha2255

### PR DESCRIPTION
**Pull request by navitha2255**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

**Review by xkrishguptaa — CHANGES_REQUESTED** [review]
Hi! You have created an incorrect profile:

1. the json syntax is invalid, you have missed quotes and some places
2. You used eddie's links and avatar, please update with your own
3. The extension for the json has a typo, it should `navitha2255.json`

File: data/navitha2255.jsomn
Comment (SaraJaoude) [inline data/navitha2255.jsomn:4]: these are no longer necessary:
```suggestion
```

File: data/navitha2255.jsomn
Comment (SaraJaoude) [inline data/navitha2255.jsomn:4]: this is not necessary:
```suggestion
  "bio":"iam a frontend developer",
```

File: data/navitha2255.jsomn
Comment (SaraJaoude) [inline data/navitha2255.jsomn:12]: closing `"` rather than `'`
```suggestion
      "icon":"FaGithubAlt"
```

File: data/navitha2255.jsomn
Comment (SaraJaoude) [inline data/navitha2255.jsomn:6]: Please replace this with your own twitter account. 

You are also missing a `"` in front of `https `

**Review by SaraJaoude — CHANGES_REQUESTED** [review]
I have made most of the changes needed for the JSON to pass, but before we can merge your PR as mentioned in another comment please

-  update the Twitter link so that it is your own
- change the file name to have `json` and the end and not `jsomn`

**Comment by SaraJaoude** [comment]
Please see my comments [here.  ](https://github.com/EddieHubCommunity/BioDrop/pull/10075/files)

If you need help with editing a PR please check out [this documentation](https://www.biodrop.io/docs/how-to-guides/editing-json) and come and chat to us on the EddieHub Discord.

**Comment by SaraJaoude** [comment]
We look forward to seeing your Profile on BioDrop, but as there is no recent activity from you to make the changes suggested I am going to close this pull request.

We hope you will raise a new pull request to create your Profile and when you do, you might want to take a look at t[hese Docs](https://www.biodrop.io/docs/quickstart-json) which will help with the suggested changes the maintainers have proposed. 

Please remember you can also create your Profile using [Forms](https://www.biodrop.io/docs/quickstart-forms).

**Pull request closed without merging**
